external-crates/move/crates/move-compiler/src/typing/ast.rs

```diff
@@ -136,7 +141,7 @@ pub struct ModuleCall {
 #[allow(clippy::large_enum_variant)]
 pub enum BuiltinFunction_ {
     Freeze(Type),
-    Assert(/* is_macro */ bool),
+    Assert(/* is_macro */ Option<Loc>),
```

Comment:
Not necessary in this PR, but would it make sense to eventually reuse the expansion infrastructure to discharge these?

Reply:
Ideally it would be nice to just deprecate this and rely on the stdlib function, but maybe we can just handroll that AST if it is missing or something